src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/KoverAndroidExtension.kt

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright 2017-2023 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.kover.gradle.plugin.dsl
+
+import org.gradle.api.*
+
+public interface KoverAndroidExtension {
+    public fun common(config: Action<KoverGeneralReportExtension>)
```

Comment:
I'm not sure that calling `variant` is a good idea, because in this block, only the report for the specified variant is configured.

Maybe `reportForAll'?